client/src/pages/AddProjectPage.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import styled from "styled-components/macro";
 import { Header } from "../components/Header";
-import { Button } from "../components/Button";
 import { Input } from "../components/Input";
 import { Navbar } from "../components/Navbar";
+import UploadImage from "../utils/uploadImage";
```

Comment:
Looks like UploadImage is a component. In this case, you could apply the naming convention for the files too (Pascal Case).

It could be in the components folder too.